src/api/preInstallApi.ts

```diff
@@ -37,5 +38,7 @@ export const getCreateMeta = async (
         )}`,
       },
     }
-  ).then((res) => res.json());
+  );
+
+  return res.json() as Promise<CreateMeta>;
```

Comment:
There is no validation to prove that this is of type CreateMeta. We should check the response code and optionally the body format.

Reply:
have been moved this functionality into the base `request`